### PR TITLE
changelog for 0.11

### PR DESCRIPTION
**Pull request by minrk**
/cc @consideRatio

### CODE REVIEW

**Comment by manics** [comment]
Are there any breaking changes in https://github.com/jupyterhub/oauthenticator/pull/323 that implementers of external Oauthenticator sub-classes need to be aware of?

File: docs/source/changelog.md [hunk 1]
```diff
@@ -8,6 +8,41 @@ command line for details.
...
+## 0.11
+
+## [0.11.0] - 2020-01-30
+
+The main change in 0.11 is a refactoring of classes to remove mixins,
+reducing the amount of boilerplate needed.
+In addition, there are some fixes to the Azure AD Authenticator.
+
+This release also *removes* the latest Authenticators added in 0.10
+(AzureAdB2COAuthenticator, AWSCognitoAuthenticator),
```
Comment (consideRatio) [inline docs/source/changelog.md]: I confirmed that YandexPassportOAuthenticator was also part of 0.10.0 and removed.

**Comment by minrk** [comment]
> Are there any breaking changes in #323 that implementers of external OAuthenticator sub-classes need to be aware of?

There aren't any known, intentional breakages here. Subclasses should be fine. One exception, perhaps if any subclasses were importing the removed mixins.

**Comment by minrk** [comment]
I added a note about subclasses possibly importing the removed mixins and the fact that env-only config should be gone.

**Comment by minrk** [comment]
@consideRatio I think so, yes!

**Comment by consideRatio** [comment]
I pushed a 0.11.0 git tag!

**Pull request merged**